packages/material-ui/src/Menu/Menu.test.js

```diff
@@ -248,4 +252,277 @@ describe('<Menu />', () => {
       );
     });
   });
+
+  describe('cascading menu', () => {
+    let clock;
+    let cascadeMount;
+    let wrapper;
+
+    before(() => {
+      clock = useFakeTimers();
+      // StrictModeViolation: uses Popover
+      cascadeMount = createMount({ strict: false });
+    });
+
+    beforeEach(() => {
+      wrapper = cascadeMount(<NestedMenu />);
```

Comment:
We should change all the references to cascading menu.

Reply:
Ah, yup